snorkel/labeling/lf/nlp.py

```diff
@@ -243,3 +224,48 @@ def __call__(self, f: Callable[..., int]) -> NLPLabelingFunction:
             disable=self.disable,
             memoize=self.memoize,
         )
+
+
+class nlp_labeling_function(base_nlp_labeling_function):
+    """Decorator to define an NLPLabelingFunction object from a function.
+
+    Parameters
+    ----------
+    name
+        Name of the LF
+    resources
+        Labeling resources passed in to ``f`` via ``kwargs``
+    pre
+        Preprocessors to run before SpacyPreprocessor is executed
+    fault_tolerant
+        Output -1 if LF execution fails?
+    text_field
+        Name of data point text field to input
+    doc_field
+        Name of data point field to output parsed document to
+    language
+        spaCy model to load
+        See https://spacy.io/usage/models#usage
+    disable
+        List of pipeline components to disable
+        See https://spacy.io/usage/processing-pipelines#disabling
+    memoize
+        Memoize preprocessor outputs?
```

Comment:
I think the statement "Whether to memoize preprocessor outputs" seems better than having a question.
Same for all other instances of memoize.

Reply:
I've seen the question format for bools quite a bit. Let's leave for now.